docs/source/guides/staking_guide.rst

```diff
@@ -0,0 +1,149 @@
+=======================
+NuCypher Staking Guide
+=======================
+
+**Ursula Staking Actions**
+
++-------------------+-------------------------------------------------------------------------+
+| Action            |  Description                                                            |
++===================+=========================================================================+
+|  `stake`          | Initialize or view NuCpher stakes (used with `--value` and `--duration`)|
++-------------------+-------------------------------------------------------------------------+
+|  `divide-stake`   | Divide stake, used with `--value` and `--duration`                      |
++-------------------+-------------------------------------------------------------------------+
+|  `collect-reward` | Collect staking reward (Policy reward collection in future PR)          |
++-------------------+-------------------------------------------------------------------------+
+
+
+**Ursula Staking Options**
+
++----------------+----------------------------------------+
+| Option         |  Description                           |
++================+========================================+
+|  `--value`     | Stake value                            |
++----------------+----------------------------------------+
+|  `--duration`  | Stake duration of extension            |
++----------------+----------------------------------------+
+|  `--index`     | Stake index                            |
++----------------+----------------------------------------+
+|  `--list`      | List stakes (used with `stake` action) |
++----------------+----------------------------------------+
+
+
+
+Interactive Method
+------------------
+
+.. code:: bash
+
+    (nucypher)$ nucypher ursula stake
+
+
+*Initial a new stake*
+
+.. code:: bash
+
+    Stage a new stake? [y/N]: y
+
+    Current balance: 10000
+    Enter stake value in NU [1500]: 3000
```

Comment:
The example may need to be changed since current minimum is 15000 NU.

Reply:
:heavy_check_mark: